language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/docs/DocAttachmentInfo.java

```diff
@@ -139,7 +139,10 @@ public String getDocumentationString(boolean newlineAtEnd) {
 
         if (deprecatedDesc != null) {
             result.append(String.format("%s# # Deprecated%n", padding));
-            result.append(String.format("%s# %s%n", padding, deprecatedDesc));
+            String[] deprecatedDescLines = deprecatedDesc.trim().split("\n");
```

Comment:
Shall we use the Platform specific `System.lineSeparator()`? There is already a constant in `CommonUtil`